tests/backends/build_system/functional/test_lock_files.py

```diff
@@ -144,6 +143,7 @@ def test_all_lock_files_are_generated_by_expected_python_version():
         "platforms we need to test are mac and Windows."
     ),
 )
+@pytest.mark.xfail(reason="pip-tools compatibility issue needs to be resolved")
```

Comment:
We should either add a comment or info in the reason linking to the upstream issue. That way we can quickly check if it's closed/fixed in the future.